core/src/main/java/org/jdbi/v3/core/Handle.java

```diff
@@ -85,6 +91,10 @@ public Connection getConnection() {
         return this.connection;
     }
 
+    public StatementBuilder getStatementBuilder() {
+        return statementBuilder;
+    }
```

Comment:
Previously, Handle had access to the constructors of Batch, Call, PreparedBatch, Query, and Update.

Since those classes are now in the `statement` package, I added a public static factory method to each, that accepts this Handle as a parameter. Making the statement builder public allowed each factory method to only need a Handle and sql parameter.